cloud_governance/cloud_resource_orchestration/common/abstract_monitor_tickets.py

```diff
@@ -192,13 +192,13 @@ def _monitor_ticket_duration(self, ticket_id: str, region_name: str, duration: i
             self.__postfix.send_email_postfix(to=user, cc=cc, subject=subject, content=body, mime_type='html',
                                               message_type=message_type)
 
-    def get_budget_exceed_alert_times(self, user: str):
+    def get_budget_exceed_alert_times(self, user: str, remaining_budget: int):
         """
-        This method returns the number of times alerts send to an user
+        This method returns the number of times alerts send to a user
         :return:
         """
         current_date = datetime.now(timezone.utc).date()
-        start_date = current_date - timedelta(days=10)
+        start_date = current_date - timedelta(days=15)
```

Comment:
Why we need to increase the days to 15 ?

Reply:
To reduce the mailing aggressiveness, send mail every 2 weeks.

Reply:
ok